.env

```diff
@@ -0,0 +1 @@
+REACT_APP_CONCERTS_API=http://localhost:4000/concerts
```

Comment:
This shouldn't be required if you use the proxy.
Your solution will not work in production (like on Heroku).